configs/example/arm/starter_se.py

```diff
@@ -203,6 +203,19 @@ def main():
         default="stdoutput",
         help="Destination for the Tarmac trace output. [Default: stdoutput]",
     )
+    parser.add_argument(
```

Comment:
I won't block this PR because of this change, but generally I am against adding more command line parameters. I also don't like the idea of being able to set any parameter via the command line. While I understand that it is useful for debugging, it makes reproducible science/experiments essentially impossible. 

I would prefer this not be exposed in any of the "starter" or "example" scripts.

Reply:
I understand your point. However, it's common practice for other configuration scripts (see fs_bigLITTLE.py) to have this option, and it would simplify our lives.

Reply:
As I said, I won't block this, because, as you said, it's pervasive. However, I do want you to know where I'm coming from.

The question to me is "who is this script for?" Given that it is called "starter_se.py" that would lead me to believe that it is for new gem5 users to "get started" using Arm. The name would imply that if you are using gem5 for research *you should not use the `starter_se.py` script*.

So, unless I'm misinterpreting the purpose of this script, the goal of this script would not be to make our (the developers') lives easier, but to make it easy to understand and get started with gem5. I think that exposing every possible parameter on the command line is not the right way to expose new users to gem5. Exposing parameters this way encourages people to use gem5 in an unreproducible and error-prone way.